indexer/feature.cpp

```diff
@@ -493,7 +495,8 @@ void FeatureType::GetPreferredNames(string & defaultName, string & intName) cons
     return;
 
   ParseCommon();
-  ::GetPreferredNames(GetID(), GetNames(), defaultName, intName);
+  auto const deviceLang = StringUtf8Multilang::GetLangIndex(languages::GetCurrentNorm());
```

Comment:
Either rename arguments (defaultName to primary and intName to secondary, and check that all call-places are fine), or keep old behavior.